mne/preprocessing/infomax_.py

```diff
@@ -23,6 +23,7 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
     ----------
     data : np.ndarray, shape (n_samples, n_features)
         The data to unmix.
+        .. note:: The data should be whitened prior to performing Infomax ICA.
```

Comment:
does this render fine?

I would put this below in the notes section and write as description.

The whitened data to unmix.